RecoLocalMuon/GEMSegment/plugins/ME0SegmentAlgorithm.cc

```diff
@@ -44,21 +44,21 @@ ME0SegAlgoMM::ME0SegAlgoMM(const edm::ParameterSet& ps) : ME0SegmentAlgorithm(ps
 /* Destructor
  *
  */
-ME0SegAlgoMM::~ME0SegAlgoMM() {
+ME0SegmentAlgorithm::~ME0SegmentAlgorithm() {
 }
 
 
-std::vector<ME0Segment> ME0SegAlgoMM::run(const ME0Ensemble& ensemble, const EnsembleHitContainer& rechits) {
+std::vector<ME0Segment> ME0SegmentAlgorithm::run(const ME0Ensemble& ensemble, const EnsembleHitContainer& rechits) {
 
   theEnsemble = ensemble;
```

Comment:
@davidlange6 - I've done all the suggestions apart from this.
Do you want me to remove the global variable theEnsemble and just pass it to the functions that use this object?